src/Simplex/Chat/Mobile.hs

```diff
@@ -48,7 +49,7 @@ cChatRecvMsg cc = deRefStablePtr cc >>= chatRecvMsg >>= newCAString
 mobileChatOpts :: ChatOpts
 mobileChatOpts =
   ChatOpts
-    { dbFilePrefix = "simplex_v1", -- two database files will be created: simplex_v1_chat.db and simplex_v1_agent.db
+    { dbFilePrefix = undefined,
```

Comment:
why is that? I am fighting `undefined`s in config, it's a bad hack, there should be a better way